src/pages/transactions/components/transaction-view.vue

```diff
@@ -537,10 +538,13 @@ export default {
       if (!this.addingComponent) {
         this.addingComponent = true
         const tempHash = [...Array(8)].map(() => Math.floor(Math.random() * 16).toString(16)).join('')
-        let d = new Date()
-        let month = d.getUTCMonth() + 1
-        let day = d.getUTCDate()
-        let date = `${d.getUTCFullYear()}/${(month < 10 ? '0' + month : month)}/${(day < 10 ? '0' + day : day)}`
+        let date = this.transaction.value.date
```

Comment:
Did we validating the transaction does not have a Date before set new one?

Reply:
In the next line validate if the date variable is undefined or null I set a new one